2020-02-07 19:38:20,200 ERROR: Exception on /search [GET] [in C:\Users\a_smi\PycharmProjects\microblog\venv\lib\site-packages\flask\app.py:1891]
Traceback (most recent call last):
  File "C:\Users\a_smi\PycharmProjects\microblog\venv\lib\site-packages\flask\app.py", line 2446, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Users\a_smi\PycharmProjects\microblog\venv\lib\site-packages\flask\app.py", line 1951, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Users\a_smi\PycharmProjects\microblog\venv\lib\site-packages\flask\app.py", line 1820, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "C:\Users\a_smi\PycharmProjects\microblog\venv\lib\site-packages\flask\_compat.py", line 39, in reraise
    raise value
  File "C:\Users\a_smi\PycharmProjects\microblog\venv\lib\site-packages\flask\app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Users\a_smi\PycharmProjects\microblog\venv\lib\site-packages\flask\app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "C:\Users\a_smi\PycharmProjects\microblog\venv\lib\site-packages\flask_login\utils.py", line 261, in decorated_view
    return func(*args, **kwargs)
  File "C:\Users\a_smi\PycharmProjects\microblog\app\main\routes.py", line 145, in search
    if total > page * current_app.config['POSTS_PER_PAGE'] else None
TypeError: '>' not supported between instances of 'dict' and 'int'
2020-02-07 19:43:08,964 INFO: Microblog startup [in C:\Users\a_smi\PycharmProjects\microblog\app\__init__.py:77]
2020-02-07 19:43:12,030 ERROR: Exception on /search [GET] [in C:\Users\a_smi\PycharmProjects\microblog\venv\lib\site-packages\flask\app.py:1891]
Traceback (most recent call last):
  File "C:\Users\a_smi\PycharmProjects\microblog\venv\lib\site-packages\flask\app.py", line 2446, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Users\a_smi\PycharmProjects\microblog\venv\lib\site-packages\flask\app.py", line 1951, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Users\a_smi\PycharmProjects\microblog\venv\lib\site-packages\flask\app.py", line 1820, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "C:\Users\a_smi\PycharmProjects\microblog\venv\lib\site-packages\flask\_compat.py", line 39, in reraise
    raise value
  File "C:\Users\a_smi\PycharmProjects\microblog\venv\lib\site-packages\flask\app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Users\a_smi\PycharmProjects\microblog\venv\lib\site-packages\flask\app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "C:\Users\a_smi\PycharmProjects\microblog\venv\lib\site-packages\flask_login\utils.py", line 261, in decorated_view
    return func(*args, **kwargs)
  File "C:\Users\a_smi\PycharmProjects\microblog\app\main\routes.py", line 145, in search
    if int(total) > page * current_app.config['POSTS_PER_PAGE'] else None
TypeError: int() argument must be a string, a bytes-like object or a number, not 'dict'
2020-02-07 19:44:05,025 INFO: Microblog startup [in C:\Users\a_smi\PycharmProjects\microblog\app\__init__.py:77]
2020-02-07 19:44:05,701 ERROR: Exception on /search [GET] [in C:\Users\a_smi\PycharmProjects\microblog\venv\lib\site-packages\flask\app.py:1891]
Traceback (most recent call last):
  File "C:\Users\a_smi\PycharmProjects\microblog\venv\lib\site-packages\flask\app.py", line 2446, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Users\a_smi\PycharmProjects\microblog\venv\lib\site-packages\flask\app.py", line 1951, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Users\a_smi\PycharmProjects\microblog\venv\lib\site-packages\flask\app.py", line 1820, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "C:\Users\a_smi\PycharmProjects\microblog\venv\lib\site-packages\flask\_compat.py", line 39, in reraise
    raise value
  File "C:\Users\a_smi\PycharmProjects\microblog\venv\lib\site-packages\flask\app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Users\a_smi\PycharmProjects\microblog\venv\lib\site-packages\flask\app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "C:\Users\a_smi\PycharmProjects\microblog\venv\lib\site-packages\flask_login\utils.py", line 261, in decorated_view
    return func(*args, **kwargs)
  File "C:\Users\a_smi\PycharmProjects\microblog\app\main\routes.py", line 145, in search
    if total > page * current_app.config['POSTS_PER_PAGE'] else None
TypeError: '>' not supported between instances of 'dict' and 'int'
2020-02-07 19:49:03,290 INFO: Microblog startup [in C:\Users\a_smi\PycharmProjects\microblog\app\__init__.py:77]
